pkg/db/congestion.go

```diff
@@ -0,0 +1,36 @@
+package db
+
+import (
+	"context"
+
+	"github.com/xmtp/xmtpd/pkg/db/queries"
+)
+
+// Gets the congestion for the minute specified by `endMinute` and the previous 4 minutes
+// returned in descending order with missing values filled with 0
+func Get5MinutesOfCongestion(
+	ctx context.Context,
+	querier *queries.Queries,
+	originatorID, endMinute int32,
+) (out [5]int32, err error) {
```

Comment:
nit: In the `CalculateCongestion(last5Minutes [5]int64, targetRatePerMinute int64)` we're working with int64, and here we're returning [5]int32.
While not strictly necessary, we could define a type alias to standardize how to handle these last 5 minutes.

Reply:
Nevermind about this. I saw it fixed in the next PR.

Reply:
Ya. I realized the same thing.

The codebase has so many int types because Postgres doesn't have a `uint` option, so anything that touches the database becomes an `int32` or `int64`